modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -1010,7 +1010,13 @@ public abstract class Base${schemaName}ResourceImpl
 								try {
 						</#if>
 
-						${deleteByIdJavaMethodSignature.methodName}(${schemaVarName}.${getterMethodName}());
+						${deleteByIdJavaMethodSignature.methodName}(${schemaVarName}.${getterMethodName}()
+							<#list deleteByIdJavaMethodSignature.javaMethodParameters as javaMethodParameter>
+								<#if freeMarkerTool.isQueryParameter(javaMethodParameter, deleteByIdJavaMethodSignature.operation)>
+									, (${javaMethodParameter.parameterType}) parameters.get("${javaMethodParameter.parameterName}")
```

Comment:
Please, apply the existing pattern (there is a macro for this use case, you can check the other `VulcanBatchEngineTaskItemDelegate` implementations)